engine/lite/request-response/app/src/test/scala/pl/touk/nussknacker/engine/requestresponse/http/RequestResponseHttpJsonSchemaSpec.scala

```diff
@@ -96,6 +96,24 @@ class RequestResponseHttpJsonSchemaSpec extends RequestResponseHttpTest {
     }
   }
 
+  it should "should handle additionalProperties without required" in {
```

Comment:
You don't have to do full request to test determiner. I think we should write unit test only for JsonSchemaSubclassDeterminer.

Reply:
That's a great idea, we should do it in the next MR :)